### DISCUSSION TITLE
--interactive not doing anything [cast wallet import defaultKey --interactive]

### DISCUSSION THREAD

**Discussion by blountdj** [Cyfrin/foundry-full-course-cu#2593] [Q&A] [answered]
I'm on lesson 21 'Never Use A Env File' of the Foundry Fundamental course.

I'm following the video and entering the following code into my terminal:

```solidity
cast wallet import defaultKey --interactive
```

After pressing enter I am expecting the 'Enter private key:' prompt to appear. But nothing happens, the cursor just goes to the next line without running anything. It doesn't seem to like the '--interactive' flag for some reason.

Not sure what is happening, tried different combinations and following the '--help' but always get the same result. Everything else has worked ok up to this point.

Can anybody help?

Edit:
Nothing also happens if I enter:
```solidity
cast wallet import --interactive defaultKey
```

Cast version = 0.2.0

**Comment by EngrPips** [comment]
Hello @blountdj, run `foundryup` then retry let's see if there is any difference.

**Comment by blountdj** [comment]
Hi @EngrPips, thanks, tried it but no luck, still the same. I also re-install forge but still had the same issue.

**Reply by EngrPips** [reply]
Please show your terminal when you run the process so we see what is going on.

**Reply by blountdj** [reply]
[5ac5cf39-da4d-4aa0-a0fd-7df660455107.webm](https://github.com/user-attachments/assets/4010aa99-836c-412c-bd8d-f05d84135f94)

Here is a video of the terminal showing what happens. 

I enter the command, press enter, wait, nothing happens, then I press CTRL+C to end.

**Reply by EngrPips** [reply]
Yeah, I see. I am unsure what is wrong. Please check the official Foundry repo https://github.com/foundry-rs/foundry to see if any similar issue has been reported and how it is solved.

**Reply by blountdj** [reply, marked as answer]
I managed to get WSL working and it ran on there no problems. No sure why but BASH didn't like it

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.
